src/lib/dev-environment/dev-environment-core.ts

```diff
@@ -761,8 +761,8 @@ async function maybeUpdateWordPressImage( slug: string ): Promise< boolean > {
 	const confirm = await prompt( {
 		type: 'select',
 		name: 'upgrade',
-		message: 'Would You like to change the WordPress version? ',
-		choices: [ 'yes', 'no', "no (don't ask anymore)" ],
+		message: 'Would you like to upgrade WordPress? ',
+		choices: [ 'no', "no (don't ask anymore)", 'yes' ],
```

Comment:
Good idea, thanks for your contribution! Only thing I'd do is add the `no (don't ask anymore)` to the bottom so it's one less keystroke for someone who does want to upgrade WP. 🙂 
```suggestion
		choices: [ 'no', 'yes', "no (don't ask anymore)" ],
```

Reply:
Sound good :)